### PR TITLE
Show expiration for certificates in Keys List in admin-ui (#17743)

### LINKED ISSUE #17743: Show expiration date of realm keys
### Description

The realm keys are currently listed like this:
```
Algorithm  | Type | Kid                                         | Provider          | Public keys
RSA-OAEP   | RSA  | 7ShgKSatq0nK5fyXxvj-fwrXj28AZ68FzHTK2memURQ | rsa-enc-generated | [Public Key] [Certificate]
RS256      | RSA  | WvCsvncC4MtdlyCmndluWsbaAoK7MBMORiD4Ai25YyU | rsa-generated     | [Public Key] [Certificate]
```

It would be helpful to also provide "valid from" and "valid to" information and the thumbprint value of the certificate.
Valid to should be displayed in the table:
```
Algorithm  | Type | Kid                                         | Provider          | Valid to            | Public keys
RS256      | RSA  | WvCsvncC4MtdlyCmndluWsbaAoK7MBMORiD4Ai25YyU | rsa-generated     | 2030-11-27T17:42:02 | [Public Key] [Certificate]
```
Other details could be shown when clicking on the [Certificate] button.

Current workaround for the missing feature:  

- Click [Certificate] button
- Copy the displayed key into a file `cert.pem`
- Add line -----BEGIN CERTIFICATE----- as first line and -----END CERTIFICATE----- as last line
- Use openssl to parse the certificate: `openssl x509 -in cert.pem -text`

### Discussion

PR which also adds this feature at old admin console as a side effect:
[KEYCLOAK-17609: OIDC broker x5t jws header #7890](https://github.com/keycloak/keycloak/pull/7890)

Realm key related discussions
[Automatic realm key rotation #13776](https://github.com/keycloak/keycloak/discussions/13776)
[Realm key(s) uses - Allowing multivalue fields #8523](https://github.com/keycloak/keycloak/discussions/8523)

### Motivation

It is important to see when a key will expire.

### Details

Keycloak Admin API Backend needs to provide additional data like _from_, _to_, _thumbprint_ with the /keys resource.

### PR DESCRIPTION
**Pull request by thomasdarimont**
Fixes #17743

Before:
**
Screenshot from 2023-09-06 21-04-16
**

After:
Screenshot from 2023-09-06 21-37-54

### CODE REVIEW

**Review by jonkoops — DISMISSED** [review]
Looks fine to me from the front-end, if someone could give the ok on the Java-side of things that would be great.

**Comment by jonkoops** [comment]
@ssilvert could you take a look at this?

**Review by ssilvert — DISMISSED** [review]

**Comment by jonkoops** [comment]
@thomasdarimont we made some changes to the way the translations work, they are now all contained in a single file. Could you rebase your work on top of `main`?

**Comment by thomasdarimont** [comment]
Hmmm, I think I messed up the display key: `displayKey: "realm-settings:validTo"`

File: js/apps/admin-ui/src/realm-settings/keys/KeysListTab.tsx [hunk 1]
```diff
@@ -180,6 +183,14 @@ export const KeysListTab = ({ realmComponents }: KeysListTabProps) => {
             cellFormatters: [emptyFormatter()],
             transforms: [cellWidth(10)],
           },
+          {
+            name: "validTo",
+            displayKey: "realm-settings:validTo",
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm-settings/keys/KeysListTab.tsx:188]:
```suggestion
            displayKey: "validTo",
```

**Review by jonkoops — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**